### DISCUSSION TITLE
Monitoring tomcat

### DISCUSSION THREAD

**Discussion by pramod-desai-minutus** [apache/skywalking#10445] [Q&A]
Hi,

I am trying to monitor  tomcat9 using apache skywalkig and I have done the following configurations,

1. Downloaded the java agent
2. Added the application name and  collector backend (IP of skywalking with Port 11800)
3. copied the skywalking-agent.jar to $TOMCAT_HOMEDIR/lib/skywalking-agent.jar
4. added same path as JAVA_OPTS in catalina.sh
5. restarted tomcat and skywalking

Both services are running ,But Nothing is showing in skywalking UI

Any suggestions would be much appreciated.

Thanks in advance!!

**Comment by wu-sheng** [comment]
Please follow setup doc. We never mentioned copy a single jar, and place it inside Tomcat.
